src/app/game-state/main-loop.service.ts

```diff
@@ -143,11 +147,7 @@ export class MainLoopService {
           }
           this.tickCount = 0;
         }
-        // if the game is inactive for longer periods of time the banked ticks might go negative.
-        if (this.bankedTicks < 0) {
-          this.bankedTicks = 0;
-          this.useBankedTicks = false;
-        }
+        this.tickCount += inactiveTickHolder;
```

Comment:
Any risk of this picking up a huge number that would bog it down?

Reply:
It simply can't due to the time limiter on the loop, if it goes over it just adds it to the bankedticks, actually tested and got it to do that.